README.md

```diff
@@ -4,7 +4,19 @@ A minimal, customizable, ready-to-compile boilerplate for Game Boy RGBDS project
 
 ## Downloading
 
-You can simply clone the repository using Git, or if you just want to download this, click the `Clone or download` button up and to the right of this. This repo is also usable as a GitHub template for creating new repositories.
+You can simply clone this repository and all its submodules required to its proper working:
+
+```sh
+git clone --recurse-submodules https://github.com/ISSOtm/gb-boilerplate.git
+```
+
+Alternatively,  if you just want to download this, click the `Clone or download` button up and to the right of this. This repo is also usable as a GitHub template for creating new repositories.
```

Comment:
Hmm, problem is that the `Download ZIP` button does *not* include submodules. How should that be handled?

Reply:
Yeah I have thought about it because ideally when using a template, one might not necessarily discover correctly how to properly clone.
I see a couple alternate solutions to this question:
* Not mentionning the download part, or warn to not use it as submodules won't be embedded
* Vendor the desired version of `hardware.inc` instead of making it a submodule

I prefer the vendoring solution as it removes a bunch of headaches. I could add a script to update the file to its latest version for ease of maintenance.

Reply:
I guess vendoring should be fine, yeah.